services/bots/src/github-webhook/handlers/blocking_labels.ts

```diff
@@ -9,6 +9,9 @@ export const LabelsToCheck: {
   [HomeAssistantRepository.CORE]: {
     'awaiting-frontend': { message: 'This PR is awaiting changes to the frontend' },
   },
+  [HomeAssistantRepository.FRONTEND]: {
+    'Backend Change Required': { message: 'This PR is awaiting changes to the backend' },
```

Comment:
The frontend have 2 labels that is currently in used.
I have asked @bramkragten which one should be blocking.

Reply:
```suggestion
    'wait for backend': { message: 'This PR is awaiting changes to the backend' },
```